Upstream: no
Reason: mariadb headers are located in a subfolder, so fix includes to use that
as well.
diff --git a/src/modules/db_mysql/km_db_mysql.c b/src/modules/db_mysql/km_db_mysql.c
index 8193588..031f8c2 100644
--- a/src/modules/db_mysql/km_db_mysql.c
+++ b/src/modules/db_mysql/km_db_mysql.c
@@ -38,7 +38,7 @@
 #include "km_dbase.h"
 #include "km_db_mysql.h"
 
-#include <mysql.h>
+#include <mysql/mysql.h>
 
 unsigned int db_mysql_timeout_interval = 2;   /* Default is 6 seconds */
 unsigned int db_mysql_auto_reconnect = 1;     /* Default is enabled   */
diff --git a/src/modules/db_mysql/km_dbase.c b/src/modules/db_mysql/km_dbase.c
index eaff327..7a6895f 100644
--- a/src/modules/db_mysql/km_dbase.c
+++ b/src/modules/db_mysql/km_dbase.c
@@ -33,8 +33,8 @@
 
 #include <stdio.h>
 #include <string.h>
-#include <mysql.h>
-#include <errmsg.h>
+#include <mysql/mysql.h>
+#include <mysql/errmsg.h>
 #include "../../core/mem/mem.h"
 #include "../../core/dprint.h"
 #include "../../core/async_task.h"
diff --git a/src/modules/db_mysql/km_my_con.c b/src/modules/db_mysql/km_my_con.c
index bed21f9..543860c 100644
--- a/src/modules/db_mysql/km_my_con.c
+++ b/src/modules/db_mysql/km_my_con.c
@@ -28,7 +28,7 @@
 
 #include "km_my_con.h"
 #include "km_db_mysql.h"
-#include <mysql.h>
+#include <mysql/mysql.h>
 
 /* MariaDB exports MYSQL_VERSION_ID as well, but changed numbering scheme */
 #if MYSQL_VERSION_ID > 80000 && ! defined MARIADB_BASE_VERSION
diff --git a/src/modules/db_mysql/km_my_con.h b/src/modules/db_mysql/km_my_con.h
index 1c79937..616fdde 100644
--- a/src/modules/db_mysql/km_my_con.h
+++ b/src/modules/db_mysql/km_my_con.h
@@ -34,7 +34,7 @@
 #include "../../lib/srdb1/db_id.h"
 
 #include <time.h>
-#include <mysql.h>
+#include <mysql/mysql.h>
 
 
 struct my_con {
diff --git a/src/modules/db_mysql/km_res.c b/src/modules/db_mysql/km_res.c
index d810dec..dadc0df 100644
--- a/src/modules/db_mysql/km_res.c
+++ b/src/modules/db_mysql/km_res.c
@@ -30,7 +30,7 @@
 
 
 #include <string.h>
-#include <mysql.h>
+#include <mysql/mysql.h>
 #include "../../lib/srdb1/db_res.h"
 #include "../../core/mem/mem.h"
 #include "../../core/dprint.h"
diff --git a/src/modules/db_mysql/km_val.h b/src/modules/db_mysql/km_val.h
index e68c831..0a9151f 100644
--- a/src/modules/db_mysql/km_val.h
+++ b/src/modules/db_mysql/km_val.h
@@ -29,7 +29,7 @@
 #ifndef KM_VAL_H
 #define KM_VAL_H
 
-#include <mysql.h>
+#include <mysql/mysql.h>
 #include "../../lib/srdb1/db_val.h"
 #include "../../lib/srdb1/db.h"
 
diff --git a/src/modules/db_mysql/my_cmd.c b/src/modules/db_mysql/my_cmd.c
index c12d649..0401db0 100644
--- a/src/modules/db_mysql/my_cmd.c
+++ b/src/modules/db_mysql/my_cmd.c
@@ -47,8 +47,8 @@
 #include <stdio.h>
 #include <time.h>  /*strptime, XOPEN issue must be >=4 */
 #include <string.h>
-#include <errmsg.h>
-#include <mysqld_error.h>
+#include <mysql/errmsg.h>
+#include <mysql/mysqld_error.h>
 
 #define STR_BUF_SIZE 1024
 
diff --git a/src/modules/db_mysql/my_cmd.h b/src/modules/db_mysql/my_cmd.h
index dea1ace..b3874e6 100644
--- a/src/modules/db_mysql/my_cmd.h
+++ b/src/modules/db_mysql/my_cmd.h
@@ -24,7 +24,7 @@
 
 #include "../../lib/srdb2/db_drv.h"
 #include "../../lib/srdb2/db_cmd.h"
-#include <mysql.h>
+#include <mysql/mysql.h>
 #include <stdarg.h>
 
 typedef enum my_flags {
diff --git a/src/modules/db_mysql/my_con.h b/src/modules/db_mysql/my_con.h
index ea6d5d7..e09b8a4 100644
--- a/src/modules/db_mysql/my_con.h
+++ b/src/modules/db_mysql/my_con.h
@@ -27,7 +27,7 @@
 #include "../../lib/srdb2/db_uri.h"
 
 #include <time.h>
-#include <mysql.h>
+#include <mysql/mysql.h>
 
 enum my_con_flags {
 	MY_CONNECTED = 1
diff --git a/src/modules/db_mysql/my_fld.h b/src/modules/db_mysql/my_fld.h
index c247e48..0246bd4 100644
--- a/src/modules/db_mysql/my_fld.h
+++ b/src/modules/db_mysql/my_fld.h
@@ -28,7 +28,7 @@
 
 #include "../../lib/srdb2/db_drv.h"
 #include "../../lib/srdb2/db_fld.h"
-#include <mysql.h>
+#include <mysql/mysql.h>
 
 /* MariaDB exports MYSQL_VERSION_ID as well, but changed numbering */
 #if MYSQL_VERSION_ID > 80000 && ! defined MARIADB_BASE_VERSION
diff --git a/src/modules/db_mysql/my_res.c b/src/modules/db_mysql/my_res.c
index ff6618c..2f09fa4 100644
--- a/src/modules/db_mysql/my_res.c
+++ b/src/modules/db_mysql/my_res.c
@@ -27,7 +27,7 @@
 #include "../../core/dprint.h"
 #include "../../lib/srdb2/db_gen.h"
 
-#include <mysql.h>
+#include <mysql/mysql.h>
 
 
 void my_res_free(db_res_t* res, struct my_res* payload)
